javascript-local-storage/data.js

```diff
@@ -0,0 +1,9 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJSON);
```

Comment:
you want to make sure that the variable does not contain a null value before you assign it to the `todos` variable.